apps/transport/lib/transport/gtfs_data.ex

```diff
@@ -53,10 +53,13 @@ defmodule Transport.GTFSData do
       |> join(:left, [gs, di, rh, r], r in DB.Resource, on: rh.resource_id == r.id)
       |> join(:left, [gs, di, rh, r, d], d in DB.Dataset, on: r.dataset_id == d.id)
       |> select([gs, di, rh, r, d], %{
-        d_id: gs.data_import_id,
-        d_title: d.custom_title,
         stop_id: gs.stop_id,
         stop_name: gs.stop_name,
+        di_id: gs.data_import_id,
```

Comment:
Ce champ est très abrégé par rapport aux autres.

Reply:
Oui c'était volontaire (de mémoire) - sur une vue détaillée au max, ça rajouterai 180k de data vu que c'est du JSON envoyé:

```
("data_import_id".length - "di_id".length) * 20_000
=> 180000
```

Reply:
Ok, je revert entièrement ça

Reply:
Du coup, question : on a `stop_location_type` comme nom de champ aussi pour la carte, est-ce que ça serait pas pertinent de le raccourcir, du genre `loc_type` ?

Reply:
Yes why not si ça ne casse rien côté JS ! Mais je ne pense pas que ça posera de souci, bonne idée !

Dans le futur si et quand on veut continuer à améliorer cette carte, il faudra voir qu'est-ce qui est lazy-loadé à côté etc de toute façon, l'idée là c'est de rester à une perf acceptable sans dégradation.

Reply:
En follow-up il faudra sûrement refaire un export du jeu "arrêt" à un moment, enrichi avec ce qui a été mis ici !

Reply:
Je suis d'accord avec toi sur l'amélioration des données dans la carte elle-même, mais il ne faut pas que ça se traduise par une augmentation de la payload.

Une stratégie que je te propose est: des champs réduits dans la payload, mais au moment de l'affichage on peut les remapper à la volée en JS sans souci, pour que ça soit plus clair.

On pourrait même intégrer de la documentation à ce moment là.

Voilà à rediscuter, comme ça on aurait la perf _et_ la lisibilité (point sur lequel je suis 100% d'accord avec toi et qui devait être rediscuté à un moment de toute façon).

Reply:
(on en reparle si ça n'est pas clair ! l'idée c'est que au hover, une transformation rapide soit faite pour afficher quelque chose de moins "beta" !)

Reply:
> Une stratégie que je te propose est: des champs réduits dans la payload, mais au moment de l'affichage on peut les remapper à la volée en JS sans souci, pour que ça soit plus clair.

Je suis d’accord que c’est la bonne chose à faire mais là tout de suite je ne veux pas me lancer là dedans.

>  il ne faut pas que ça se traduise par une augmentation de la payload.

Quand tu vois le contenu (long) des champs d_name et stop_name, rajouter deux caractères aux noms des clefs c’est minime dans la taille de la payload par rapport au contenu total, d’autant plus que c’est compressé.

Bon eh bien dans ce cas là, je réduis la taille de la clef `stop_location_type` en `location_type`, et je mets `di_id` et `ds_name`, hop, on gagne en taille. Ça te va ? D’ailleurs, c’est cohérent avec la norme GTFS de mettre `location_type` plutôt que `stop_location_type`, voir  https://developers.google.com/transit/gtfs/reference?hl=fr#stopstxt

Reply:
> Je suis d’accord que c’est la bonne chose à faire mais là tout de suite je ne veux pas me lancer là dedans.

Je comprends très bien pas de souci.

> Quand tu vois le contenu (long) des champs d_name et stop_name, rajouter deux caractères aux noms des 
> clefs c’est minime dans la taille de la payload par rapport au contenu total, d’autant plus que c’est compressé.

Oui je fais juste attention au fait que ça ne soit pas un argument pour recommencer dans deux semaines, et qu'avec le temps ça devienne une dégradation qui est plus compliquée à gérer etc (cf "Broken Window").

Même compressé il y a du volume de traitement côté serveur sur de "grosses maps" qui peut être coûteux en Elixir (et dépendre de la taille des clés aussi), et idem au niveau "demarshalling" côté client.

Faisons "clair pour l'utilisateur dans l'API" pour ce tour, et ajuste comme tu le sens sur la carte pour ce tour.

Il faut juste pas que ça continue dans cette voie là derrière comme tendance générale, et qu'on repasse dessus (ex: créer un ticket pour penser à re-traiter ça, spécifique à la carte).

Reply:
Pardon pour clarifier -> remettre un ticket qui indique qu'il faudra faire attention à la taille de la payload, et noter quelques éléments dedans, pour ne pas que ça dérape trop.